docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+version: '3.7'
+
+services:
+  db:
+    image: postgres:12.4
+    restart: always
+    volumes:
+    - db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: diaper
+      POSTGRES_PASSWORD: diaper
+      POSTGRES_DB: dummy
+    networks:
+      net: {}
+
+  app:
+    build:
+      context: .
+    restart: always
+    ports:
+      - 3000:3000
+    environment:
+      RAILS_ENV: development
+      DATABASE_URL: "postgresql://diaper:diaper@db/diaper"
+    networks:
+      net: {}
+    depends_on:
+    - db
+    volumes:
+    - code:/opt/diaper
+
+volumes:
+  db:
+  code:
+
+networks:
+  net:
+    driver: bridge
```

Comment:
Is this necessary for Mutagen support, or something else?

Reply:
Nope. Not needed. I added this while I was trying to debug something else and forgot to remove. The default behaviour is the one listed there. Already removed it via https://github.com/rubyforgood/diaper/pull/1893/commits/3416e5e6a3a78a52ffe38620f309ec8a31c52ecd.